decidim-participatory_processes/spec/commands/update_participatory_process_spec.rb

```diff
@@ -5,11 +5,10 @@
 module Decidim::ParticipatoryProcesses
   describe Admin::UpdateParticipatoryProcess do
     describe "call" do
-      let(:my_process) { create(:participatory_process) }
-      let!(:taxonomizations) do
-        2.times.map { create(:taxonomization, taxonomy: create(:taxonomy, :with_parent, organization: my_process.organization), taxonomizable: my_process) }
-      end
-      let(:taxonomy) { create(:taxonomy, :with_parent, organization: my_process.organization) }
+      let(:organization) { create(:organization) }
+      let(:my_process) { create(:participatory_process, organization:, taxonomies:) }
+      let(:taxonomies) { create_list(:taxonomy, 2, :with_parent, organization:) }
+      let(:taxonomy) { create(:taxonomy, :with_parent, organization:) }
```

Comment:
le'ts keep it as symilar to the conference update command. 
```suggestion
      let(:taxonomy) { create(:taxonomy, :with_parent, organization:) }
      let(:user) { create(:user, :admin, :confirmed, organization:) }
```

Reply:
ok